### PR TITLE
Remove app-model CommandId enum in favor of plain strings

### LINKED ISSUE #6325: Use consistent separator for `commandID`s
Napari app model command IDs all use `:` between words, e.g., see:

https://github.com/napari/napari/blob/1b1baa8291ff03fb7732b9ca27b33ef471da0b71/napari/_app_model/constants/_commands.py#L18-L28

For plugins we use the command `id` field for this. Our builtins use `.` instead, e.g.,:

https://github.com/napari/napari/blob/1b1baa8291ff03fb7732b9ca27b33ef471da0b71/napari_builtins/builtins.yaml#L10-L15

Can/should we change these over?

Note I know we can't dictate what other plugins do but should we be consistent within napari and builtins?

### PR DESCRIPTION
**Pull request by lucyleeow**
# References and relevant issues
Towards #6516
Closes #6325 ( I think we agreed on `.`)

# Description
Removes command ID's, we just use strings now. Allows us to remove the `_commands.py` file

### CODE REVIEW

**Comment by lucyleeow** [comment]
Sorry this PR may be as unpleasant to review as it was for me to work on.

**Review by Czaki — APPROVED** [review]
looks ok for me

File: napari/_qt/_qapp_model/qactions/_view.py [hunk 1]
```diff
@@ -131,20 +131,60 @@ def _get_current_tooltip_visibility() -> bool:
...
-for cmd, viewer_attr, sub_attr in (
-    (CommandId.TOGGLE_VIEWER_AXES, 'axes', 'visible'),
-    (CommandId.TOGGLE_VIEWER_AXES_COLORED, 'axes', 'colored'),
-    (CommandId.TOGGLE_VIEWER_AXES_LABELS, 'axes', 'labels'),
-    (CommandId.TOGGLE_VIEWER_AXES_DASHED, 'axes', 'dashed'),
-    (CommandId.TOGGLE_VIEWER_AXES_ARROWS, 'axes', 'arrows'),
-    (CommandId.TOGGLE_VIEWER_SCALE_BAR, 'scale_bar', 'visible'),
-    (CommandId.TOGGLE_VIEWER_SCALE_BAR_COLORED, 'scale_bar', 'colored'),
-    (CommandId.TOGGLE_VIEWER_SCALE_BAR_TICKS, 'scale_bar', 'ticks'),
+for cmd, cmd_title, viewer_attr, sub_attr in (
```
Comment (jni) [inline napari/_qt/_qapp_model/qactions/_view.py]: The start of the `for` is now separated from the body of the loop by a huge gap now, which makes this really hard to read. I suggest one or both of:
- add a comment above the for loop that says what the for-loop is testing, and/or
- make the list of tuples before the loop, then have the loop iterate over that list.

**Review by jni — APPROVED** [review]
Very happy to see this! 😍 I left a minor optional comment about one of the changes that is now a bit more difficult to read.

It would be good also to follow this up with a docs PR that has some guidance for developers on command-id naming conventions.

File: napari/_qt/_qapp_model/qactions/_view.py (hunk 1 above)
Comment (lucyleeow) [reply napari/_qt/_qapp_model/qactions/_view.py]: Good idea. Will do.

Comment (lucyleeow) [reply napari/_qt/_qapp_model/qactions/_view.py]: @jni done, thank you!

Comment (lucyleeow) [reply napari/_qt/_qapp_model/qactions/_view.py]: Ah forgot to pull before pushing. Done @jni !

**Pull request merged**